libparsec/crates/protocol/tests/authenticated_cmds/v5/invite_list.rs

```diff
@@ -5,6 +5,7 @@
 #![allow(clippy::unwrap_used)]
 
 use super::authenticated_cmds;
+use libparsec_types::prelude::*;
 
 use libparsec_tests_lite::{hex, p_assert_eq};
 use libparsec_types::{InvitationStatus, InvitationToken};
```

Comment:
can be removed since The One All True `use libparsec_types::prelude::*;` is now used \o/

Reply:
43224d8dc70ff51d3358bd6e03b216e7704abbea